samples/client/petstore-security-test/typescript-fetch/api.ts

```diff
@@ -115,11 +115,26 @@ export const FakeApiFp = {
  */
 export class FakeApi extends BaseAPI {
     /** 
-     * To test code injection *_/ &#39; \&quot; &#x3D;end \\r\\n \\n \\r
-     * @param test code inject * &#39; &quot; &#x3D;end rn n r To test code injection *_/ &#39; \&quot; &#x3D;end \\r\\n \\n \\r
+     * To test code injection *_/ &#39; \&quot; &#x3D;end -- \\r\\n \\n \\r
+     * @param test code inject * &#39; &quot; &#x3D;end  rn n r To test code injection *_/ &#39; \&quot; &#x3D;end -- \\r\\n \\n \\r
      */
-    testCodeInjectEndRnNR(params: {  test code inject * &#39; &quot; &#x3D;end rn n r?: string; }) {
-        return FakeApiFp.testCodeInjectEndRnNR(params)(this.fetch, this.basePath);
+    testCodeInjectEndRnNR(params: {  test code inject * &#39; &quot; &#x3D;end  rn n r?: string; }, options?: any) {
```

Comment:
@Vrolijkx (cc @wing328 ) nope it doesn't compile - it appears to be a more general problem whereby the security tests YAML is specified without string delimiters and has spaces, which isn't being handled by the template.

Looks like these params need be string delimited in the signature - which seems horrible style-wise when used in this typescript interface context, but actually is perfectly valid as its defining the interface for some JSON that can absolutely have spaces in a string key

To compile it should become... 
testCodeInjectEndRnNR(params: { "test code inject * &#39; &quot; &#x3D;end rn n r"?: string; }, options?: any) { return FakeApiFp.testCodeInjectEndRnNR(params, options)(fetch, basePath); }

It is a strange context for the problem to have been surfaced - I can't see why I as an API developer would introduce malicious injection code as an expected parameter key?! As it stands it wouldn't make the API vulnerable, as it wouldn't even compile in the first place! However it would be perfectly reasonable to declare to expect a JSON param with spaces as key ie. {"Full name": "Nic Cage" } so should probably be addressed anyway

I reckon this should be raised and looked at in it's own ticket, as probably merits a little more discussion - and its also broken in the other typescript modules